build/nix/fix_archives.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+#
+#
+# Gets dist/tarball_dirty created by Goreleaser (all files in root path) and reorganize files in correct path
+#
+#
+PROJECT_PATH=$1
+
+for tarball_dirty in $(find dist -regex ".*_dirty\.\(tar.gz\)");do
```

Comment:
Just a note so we don't forget to apply the script improvements in discussed in other PRs (e.g. `curl -L https://github.com/newrelic/nri-kafka/pull/92/commits/e11f9b02f00461cf4bd05fc6b1a3680226c2626e.patch | git apply -`)